mitiq/interface/mitiq_braket/tests/test_conversions_braket.py

```diff
@@ -104,6 +104,40 @@ def test_from_braket_parameterized_single_qubit_gates(qubit_index):
     )
 
 
+@pytest.mark.parametrize("qubit_index", (0, 3))
+def test_from_braket_single_qubit_two_parameter_gates(qubit_index):
+    braket_circuit = BKCircuit()
+    pgates = [
+        braket_gates.PRx,
+    ]
+
+    angles = np.random.RandomState(11).random((len(pgates), 2))
+    instructions = [
+        Instruction(rot(a[0], a[1]), target=qubit_index)
+        for rot, a in zip(pgates, angles)
+    ]
+    for instr in instructions:
+        braket_circuit.add_instruction(instr)
+    cirq_circuit = from_braket(braket_circuit)
+
+    for i, op in enumerate(cirq_circuit.all_operations()):
+        assert np.allclose(
+            instructions[i].operator.to_matrix(), protocols.unitary(op)
```

Comment:
Is there a difference between `protocols.unitary(op)` and `op.unitary()`? Maybe the `unitary` method only exists on circuits and this is an op?